config/puma.rb

```diff
@@ -21,14 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
-
-# Use the `preload_app!` method when specifying a `workers` number.
-# This directive tells Puma to first boot the application and load code
-# before forking the application. This takes advantage of Copy On Write
-# process behavior so workers use less memory.
-#
-# preload_app!
+workers ENV.fetch("WEB_CONCURRENCY") { 3 }
```

Comment:
@DeeDeeG if you could share with me some data about the workload in production would be nice, and I could help to improve this setup.

Reply:
@brunoocasali hmm, I don't have access to that data myself.

cc @mi-wood @tkwidmer do you have data about our traffic/resource usage on Heroku?

This pull request is to look at enabling multiple workers/concurrency in Puma. It would be useful to know what our performance needs are. If you have anything relevant to share from the Heroku dashboards, that would be great.

Thanks.

Reply:
If you people use any APM will be nice to see them too, like new relic, scout apm or others..